include/aspect/parameters.h

```diff
@@ -202,6 +202,109 @@ namespace aspect
      */
 
     /**
+     * @name Formulation settings
+     * @{
+     */
+    struct CombinedFormulation
```

Comment:
Some more documentation would be nice to have, both for the struct/namespace and for its members, to explain what they are used for and what they concretely do.